app/forms/idv/doc_pii_form.rb

```diff
@@ -83,7 +83,7 @@ def error_count
     end
 
     def zipcode_valid?
-      zipcode.nil? || zipcode.is_a?(String)
+      zipcode.is_a?(String)
```

Comment:
I would maybe do:
```suggestion
      zipcode.is_a?(String) && zipcode.present?
```
to make sure that empty-string zipcodes are not accepted